css/main.css

```diff
@@ -1,3 +1,89 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+:root {
+    --beige: #E6DACD;
+    --lbeige: #F4ECE6;
+    --blue: #0150FD;
+    --white: #FFF;
+    --black: #000;
+}
```

Comment:
Great work using CSS variables!

Reply:
💡 A little trick to avoid repeating the same CSS in all your CSS files: you could create a `common.css` file which you import in all your HTML pages, on top of each page's individual styles. This way you put all the common styles (such as colours, fonts etc) in one place and if you need to change them, the changes will be instantly rolled out across all pages.